ods_ci/tests/Tests/200__monitor_and_manage/200__metrics/205_bias_metrics.robot

```diff
@@ -0,0 +1,199 @@
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/ModelServer.resource
+Resource            ../../../Resources/Page/OCPDashboard/Monitoring/Metrics.robot
+Library             OpenShiftLibrary
+Suite Setup         Bias Metrics Suite Setup
+Suite Teardown      RHOSi Teardown
+
+*** Variables ***
+${DEFAULT_MONITORING_NS}=            openshift-user-workload-monitoring
+${SETUP_MONITORING}=                 ${TRUE}
+${PRJ_TITLE}=                        model-serving-project
+${MODEL_ALPHA}=                      demo-loan-nn-onnx-alpha
+${MODEL_PATH_ALPHA}=                 trusty/loan_model_alpha.onnx
+${MODEL_BETA}=                       demo-loan-nn-onnx-beta
+${MODEL_PATH_BETA}=                  trusty/loan_model_beta.onnx
+${TRUSTYAI_RESOURCEPATH}=            ods_ci/tests/Resources/Files/TrustyAI
+${TRUSTYAI_CR_FILEPATH}=             ${TRUSTYAI_RESOURCEPATH}/trustyai_crd.yaml
+${MONITORING_CONFIG_FILEPATH}=       ods_ci/tests/Resources/Files/cluster-monitoring-config.yaml
+${UWM_CONFIG_FILEPATH}=              ods_ci/tests/Resources/Files/user-workload-monitoring-config.yaml
+${aws_bucket}=                       rhods-public
+${RUNTIME_NAME}=                     Model Bias Serving Test
+${PRJ_DESCRIPTION}=                  Model Bias Project Description
+${framework_onnx}=                   onnx - 1
+
+*** Test Cases ***
+
+Verify TrustyAI Operator Installation
+    [Documentation]    Verifies that the TrustyAI operator has been
+    ...    deployed in the ${APPLICATIONS_NAMESPACE} namespace
+    [Tags]    Smoke
+    ...       Tier1    ODS-2481
+    ...       OpenDataHub
+    Run Keyword And Continue On Failure  Wait Until Keyword Succeeds  1 min  10 sec  Verify trustyai-service-operator-controller-manager Deployment
+
+Verify Bias Metrics are available in CLI for models deployed prior to enabling Trusty service
+    [Documentation]    Verifies that the Bias metrics are available in the Prometheus for a model which was
+    ...                 deployed prior to enabling the TrustyAI service
+    [Tags]    Smoke
+    ...       Tier1   ODS-2482    ODS-2479
+    ...       OpenDataHub
+    [Teardown]    Delete Data Science Project From CLI   ${PRJ_TITLE}
+    Open Model Serving Home Page
+    Serve Model    project_name=${PRJ_TITLE}    model_name=${MODEL_ALPHA}    framework=${framework_onnx}    existing_data_connection=${TRUE}
+    ...    data_connection_name=model-serving-connection    model_path=${MODEL_PATH_ALPHA}    model_server=${RUNTIME_NAME}
+    ${runtime_pod_name} =    Replace String Using Regexp    string=${RUNTIME_NAME}    pattern=\\s    replace_with=-
+    ${runtime_pod_name} =    Convert To Lower Case    ${runtime_pod_name}
+    Sleep    45s
+    ${return_code}    ${output}    Run And Return Rc And Output   oc get pod -n ${PRJ_TITLE} | grep "5/5" -o
+    Should Be Equal    ${output}   5/5
+    Should Be Equal As Integers    ${return_code}	 0
```

Comment:
you could probably replace these lines with a call similar to `Wait For Pods To Be Ready    label_selector=serving.kserve.io/inferenceservice=${model_name}
    ...    namespace=${test_namespace}`

(that label selector works for kserve, but model mesh pods might have something similar)

Reply:
or using the `Wait For Pods Status` kw